client-internal/src/main/java/org/terracotta/angela/client/AngelaOrchestrator.java

```diff
@@ -140,6 +140,10 @@ public static AngelaOrchestratorBuilder builder() {
     private Function<Agent, Executor> executorBuilder;
     private String mode; // just for toString()
 
+    public AngelaOrchestratorBuilder() {
+      igniteRemote();
+    }
```

Comment:
My idea was to prevent people using the builder constructor since it close the door to any flexible api change later regarding the builder class.

this should be:

```
private AngelaOrchestratorBuilder() {
      igniteRemote();
    }
```

And to get a builder instance:

```
AngelaOrchestratorBuilder builder = AngelaOrchestrator.builder();
```